### ISSUE TITLE
Accessibility Test: Unable to move focus to within the player

### ISSUE THREAD

**Issue by kylehuynh205** [digitalutsc/replaywebpage#2]
Turn on the screen reader: 

https://tamil2.digital.utsc.utoronto.ca/node/6

Turn on Screen reader, it's unable to change focus to component within the player:

**Comment by Natkeeran** [comment]
Old issue to reproduce.
